database/migrations/000090_entity_properties.up.sql

```diff
@@ -0,0 +1,51 @@
+-- Copyright 2024 Stacklok, Inc
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+CREATE TABLE IF NOT EXISTS entities (
+	id          UUID NOT NULL DEFAULT gen_random_uuid() PRIMARY KEY,
+	entity_type entities NOT NULL, 
+	name        TEXT NOT NULL,
+	project_id  UUID NOT NULL REFERENCES project(id),
+	provider_id UUID NOT NULL REFERENCES provider(id),
+	created_at  TIMESTAMP NOT NULL DEFAULT NOW(),
+	most_recent_evaluation TIMESTAMP, -- last time a profile/rule was run against this entity
```

Comment:
If we don't have a use for this column, should we add it already?

Reply:
We can remove it, no biggie.